.github/workflows/wheels.yml

```diff
@@ -199,3 +199,61 @@ jobs:
           source ci/upload_wheels.sh
           set_upload_vars
           upload_wheels
+
+  # Build Pyodide wheels and upload them to Anaconda.org
+  # NOTE: this job must be kept in sync with the emscripten job in unit-tests.yml.
+  # The difference is that this does not run the tests after building the wheels.
+  build_pyodide_wheels:
+    name: Build Pyodide wheels
+    # Run on schedule, manual (workflow dispatch), and push events
+    # Skip on PRs, because these are running in the unit tests job already
+    if: >-
+      (github.event_name == 'schedule') ||
+      (github.event_name == 'workflow_dispatch') &&
+      (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') && ( ! endsWith(github.ref, 'dev0')))

```

Comment:
This condition _can_ probably be skipped, because the intention of this PR is not to upload to https://anaconda.org/multibuild-wheels-staging/pandas,